README.rst

```diff
@@ -17,9 +17,10 @@ The set of outputs is open-ended and allows for your own extensions. Sysl has be
 Installation
 ------------
 If you are interested in trying out Sysl, you will need to install `Python 2.7 <https://www.python.org/downloads/>`_ .
-OSX users, we recommend installing Python 2.7 with `Homebrew <https://brew.sh/>`_ ::
+OSX users, we recommend installing Python 2.7 with `Homebrew <https://brew.sh/>`_  and adding it to the `PATH` in you `.bash_profile`::
 
   > brew install python
+  > echo export PATH=/usr/local/opt/python/libexec/bin:\$PATH >> ~/.bash_profile
```

Comment:
Why python/libexec/bin and not just python/bin? Usually you would not have a libexec directory in your path as it contains internal executable components of a package.

Reply:
See above